doc/source/examples/transformations/center_protein_in_box.ipynb

```diff
@@ -41,7 +41,7 @@
     {
```

Comment:
<p>Whoops. Raised https://github.com/MDAnalysis/mdanalysis/issues/3909</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>thanks for fixing it!</p>